Fail to cross compile to ppc64. "CONFIG_PVXS_MODULE was not loaded."

Trying to build PVXS for ppc64 but it fails with error:
```
../../cfg/RULES_PVXS_MODULE:10: *** CONFIG_PVXS_MODULE was not loaded.  Stop.
```
Here is the full log for the build 
```
make -C ./configure install
make[1]: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/configure'
perl -CSD /home/lamm/src/pvxs-vanilla-env/epics/bin/linux-x86_64/makeMakefile.pl O.linux-x86_64 ../..
perl -CSD /home/lamm/src/pvxs-vanilla-env/epics/bin/linux-x86_64/makeMakefile.pl O.linux-ppc64 ../..
mkdir -p O.Common
make -C O.linux-x86_64 -f ../Makefile TOP=../.. \
    T_A=linux-x86_64 install
make[2]: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/configure/O.linux-x86_64'
Installing CFG file ../../cfg/CONFIG_PVXS_VERSION
mkdir ../../cfg
perl -CSD /home/lamm/src/pvxs-vanilla-env/epics/bin/linux-x86_64/convertRelease.pl checkRelease
/usr/bin/gcc -x c -E  -D_GNU_SOURCE -D_DEFAULT_SOURCE         -DUSE_TYPED_RSET  -D_X86_64_ -DUNIX  -Dlinux    -I. -I../O.Common -I. -I. -I.. -I../../include/compiler/gcc -I../../include/os/Linux -I../../include -I/home/lamm/src/pvxs-vanilla-env/epics/include/compiler/gcc -I/home/lamm/src/pvxs-vanilla-env/epics/include/os/Linux -I/home/lamm/src/pvxs-vanilla-env/epics/include         ../toolchain.c > TOOLCHAIN.tmp
mv TOOLCHAIN.tmp TOOLCHAIN
make[2]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/configure/O.linux-x86_64'
make -C O.linux-ppc64 -f ../Makefile TOP=../.. \
    T_A=linux-ppc64 install
make[2]: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/configure/O.linux-ppc64'
perl -CSD /home/lamm/src/pvxs-vanilla-env/epics/bin/linux-x86_64/convertRelease.pl checkRelease
/opt/ifc14xx/1.6.0-5.4/sysroots//x86_64-fslsdk-linux/usr/bin/powerpc64-fsl-linux/powerpc64-fsl-linux-gcc -x c -E  -D_GNU_SOURCE -D_DEFAULT_SOURCE         -DUSE_TYPED_RSET  -D_ppc_64_ -D_ppc_64_ -DUNIX  -Dlinux -DUNIX  -Dlinux    -I. -I../O.Common -I. -I. -I.. -I../../include/compiler/gcc -I../../include/os/Linux -I../../include -I/home/lamm/src/pvxs-vanilla-env/epics/include/compiler/gcc -I/home/lamm/src/pvxs-vanilla-env/epics/include/os/Linux -I/home/lamm/src/pvxs-vanilla-env/epics/include         ../toolchain.c > TOOLCHAIN.tmp
mv TOOLCHAIN.tmp TOOLCHAIN
make[2]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/configure/O.linux-ppc64'
make[1]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/configure'
make -C ./setup install
make[1]: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup'
perl -CSD /home/lamm/src/pvxs-vanilla-env/epics/bin/linux-x86_64/makeMakefile.pl O.linux-x86_64 ../..
perl -CSD /home/lamm/src/pvxs-vanilla-env/epics/bin/linux-x86_64/makeMakefile.pl O.linux-ppc64 ../..
mkdir -p O.Common
make -C O.linux-x86_64 -f ../Makefile TOP=../.. \
    T_A=linux-x86_64 install
make[2]: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-x86_64'
Installing CFG file ../../cfg/CONFIG_PVXS_MODULE
Installing CFG file ../../cfg/RULES_PVXS_MODULE
perl -CSD /home/lamm/src/pvxs-vanilla-env/epics/bin/linux-x86_64/expandVars.pl  -a linux-x86_64 -t "../.." "-DLIBEVENT=" "-DLIBEVENT_BUNDLE_LIBS=event_core event_pthreads" "-DLIBEVENT_SYS_LIBS=" ../TOOLCHAIN_PVXS.target@ TOOLCHAIN_PVXS.linux-x86_64
Installing CFG file ../../cfg/TOOLCHAIN_PVXS.linux-x86_64
make[2]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-x86_64'
make -C O.linux-ppc64 -f ../Makefile TOP=../.. \
    T_A=linux-ppc64 install
make[2]: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-ppc64'
../../cfg/RULES_PVXS_MODULE:10: *** CONFIG_PVXS_MODULE was not loaded.  Stop.
make[2]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-ppc64'
make[1]: *** [/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_ARCHS:58: install.linux-ppc64] Error 2
make[1]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup'
make: *** [/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_DIRS:85: setup.install] Error 2
```

The steps I made were simply configure EPICS Base to build for ppc64 and point out the SDK utilities for the build then build both base and PVXS.  For what I understand of the build system, my suspicion is that, at this stage it should not be loading `RULES_PVXS_CONFIG`. It should still be creating `TOOLCHAIN_PVXS.linux-ppc64`.

I'm using EPICS Base 7.0 branch and PVXS master branch.

For context when running `make --debug=v` its clear that for x86_64 it is not loading `RULES_PVXS_MODULE` but it is for ppc64. 

x86_64:
```
Reading makefile '/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_FILE_TYPE' (search path) (no ~ expansion)...                                                                                                                                                                
Reading makefile '/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES.Db' (search path) (no ~ expansion)...                                                                                                                                                                       
Reading makefile '/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_EXPAND' (search path) (no ~ expansion)...                                                                                                                                                                   
Reading makefile '/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_COMMON' (search path) (no ~ expansion)...                                                                                                                                                                   
Reading makefile '/home/lamm/configure/RULES_USER' (search path) (don't care) (no ~ expansion)...                                                                                                                                                                                   
Updating makefiles....                                                                                                                                                                                                                                                              
Updating goal targets....                                                                                                                                                                                                                                                           
Considering target file 'install'.                                                                                                                                                                                                                                                  
 File 'install' does not exist.                                                                                                                                                                                                                                                     
  Considering target file 'buildInstall'.                                                                                                                                                                                                                                           
   File 'buildInstall' does not exist.                                                                                                                                                                                                                                              
    Considering target file '../../cfg/CONFIG_PVXS_MODULE'.                                                                                                                                                                                                                         
     File '../../cfg/CONFIG_PVXS_MODULE' does not exist.                                                                                                                                                                                                                            
      Considering target file '../CONFIG_PVXS_MODULE'.                                                                                                                                                                                                                              
       Finished prerequisites of target file '../CONFIG_PVXS_MODULE'.                                                                                                                                                                                                               
      No need to remake target '../CONFIG_PVXS_MODULE'.                                                                                                                                                                                                                             
     Finished prerequisites of target file '../../cfg/CONFIG_PVXS_MODULE'.                                                                                                                                                                                                          
    Must remake target '../../cfg/CONFIG_PVXS_MODULE'.                                                                                                                                                                                                                              
Installing CFG file ../../cfg/CONFIG_PVXS_MODULE                                                                                                                                                                                                                                    
    Successfully remade target file '../../cfg/CONFIG_PVXS_MODULE'.                                                                                                                                                                                                                 
    Considering target file '../../cfg/RULES_PVXS_MODULE'.                                                                                                                                                                                                                          
     File '../../cfg/RULES_PVXS_MODULE' does not exist.                                                                                                                                                                                                                             
      Considering target file '../RULES_PVXS_MODULE'.                                                                                                                                                                                                                               
       Finished prerequisites of target file '../RULES_PVXS_MODULE'.                                                                                                                                                                                                                
      No need to remake target '../RULES_PVXS_MODULE'.                                                                                                                                                                                                                              
     Finished prerequisites of target file '../../cfg/RULES_PVXS_MODULE'.                                                                                                                                                                                                           
    Must remake target '../../cfg/RULES_PVXS_MODULE'.                                                                                                                                                                                                                               
Installing CFG file ../../cfg/RULES_PVXS_MODULE
```

ppc64:
```
Reading makefile '/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_FILE_TYPE' (search path) (no ~ expansion)...                      
Reading makefile '../../cfg/RULES_PVXS_MODULE' (search path) (no ~ expansion)...                                                          
../../cfg/RULES_PVXS_MODULE:10: *** CONFIG_PVXS_MODULE was not loaded.  Stop.                                                             
make[2]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-ppc64'                                                     
make[1]: *** [/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_ARCHS:58: install.linux-ppc64] Error 2                                
make[1]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup'                                                                   
make: *** [/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_DIRS:85: setup.install] Error 2
```

> Correct, although this should not be an issue. The build in setup/ should work correctly in the presence of ../cfg/ left over from a previous successful build.
> The state which should not exist is having cfg/RULES_PVXS_CONFIG without also having cfg/CONFIG_PVXS_MODULE. The only way I can imagine this happening is momentarily during a parallel build with at least two target archs.

So there is indeed something weird happening here. If I try to check the files for x86_64 as you are doing I still get the error.
```
[lamm@Dio pvxs]$ make -C setup/O.linux-x86_64 show-makefiles                                                                                                                                                                                                                 
make: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-x86_64'                                                                                                                                                                                         
make -f ../Makefile TOP=../.. T_A=linux-x86_64  show-makefiles                                                                                                                                                                                                               
make[1]: Entering directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-x86_64'                                                                                                                                                                                      
../../cfg/RULES_PVXS_MODULE:10: *** CONFIG_PVXS_MODULE was not loaded.  Stop.                                                                                                                                                                                                
make[1]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-x86_64'                                                                                                                                                                                       
make: *** [Makefile:8: show-makefiles] Error 2                                                                                                                                                                                                                               
make: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-x86_64'
```

Indeed `CONFIG_PVXS_MODULE` is never read.

```
[lamm@Dio pvxs]$ make -d -C setup/O.linux-ppc64 show-makefiles  | grep PVXS
Reading makefile '../../cfg/RULES_PVXS_MODULE' (search path) (no ~ expansion)...
../../cfg/RULES_PVXS_MODULE:10: *** CONFIG_PVXS_MODULE was not loaded.  Stop.
```

`INSTALL_CFG` indeed does not have `$(TOP)/cfg`
```
"INSTALL_CFG = /home/lamm/src/pvxs-vanilla-env/epics/cfg"
"TOP_CFG_CONFIGS = /home/lamm/src/pvxs-vanilla-env/epics/cfg/CONFIG_CA_MODULE /home/lamm/src/pvxs-vanilla-env/epics/cfg/CONFIG_CA_VERSION /home/lamm/src/pvxs-vanilla-env/epics/cfg/CONFIG_DATABASE_MODULE /home/lamm/src/pvxs-vanilla-env/epics/cfg/CONFIG_DATABASE_VERSION /home/lamm/src/pvxs-vanilla-env/epics/cfg/CONFIG_LIBCOM_MODULE /home/lamm/src/pvxs-vanilla-env/epics/cfg/CONFIG_LIBCOM_VERSION"
../../cfg/RULES_PVXS_MODULE:10: *** CONFIG_PVXS_MODULE was not loaded.  Stop.
make[2]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup/O.linux-ppc64'
make[1]: *** [/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_ARCHS:58: install.linux-ppc64] Error 2
make[1]: Leaving directory '/home/lamm/src/pvxs-vanilla-env/pvxs/setup'
make: *** [/home/lamm/src/pvxs-vanilla-env/epics/configure/RULES_DIRS:85: setup.install] Error 2
```

--- a/configure/CONFIG
+++ b/configure/CONFIG
@@ -28,3 +28,7 @@ ifdef T_A
  -include $(TOP)/configure/O.$(T_A)/TOOLCHAIN
 endif
 
+ifneq ($(INSTALL_LOCATION),$(TOP))
+  include $(wildcard $(INSTALL_CFG)/CONFIG*)
+endif
+

